.github/workflows/aarch64.yml

```
@@ -29,7 +31,7 @@ jobs:
         uses: actions/download-artifact@v1
         with:
           name: crystal-aarch64-musl
-          path: .build/
+          path: ./
```

Can change to v2 to be able to drop the `path` argument.
Here and elsewhere

```suggestion
        uses: actions/download-artifact@v2
        with:
          name: crystal-aarch64-musl
```